src/components/button/button.tsx

```diff
@@ -87,19 +88,21 @@ export class Button
    */
   @Prop() readonly size: "large" | "normal" | "small" = "normal";
 
-  /**
-   * Emitted when the element is clicked.
-   */
-  @Event() gxClick: EventEmitter;
-
+  @Listen("click", { capture: true })
   private handleClick(event: UIEvent) {
+    event.stopPropagation();
```

Comment:
Shouldn't we stop propagation only when `disabled` is `true`?

Reply:
I suspect when control is enabled and clicked, it will bubbling/capturing...
Before it was that way... the stop propagation only when `disabled` is `true`... I don't remember why I changed it.
Btw, I will try putting the `stopPropagation`  inside a condition if `disabled` is `true`.

Reply:
@ncardeli putting the `stopPropagation` inside a condition if  `disabled` is `true`, the bubbling/capturing appears!
![image](https://user-images.githubusercontent.com/33690842/92419554-a5cd8080-f144-11ea-916f-a098a9b80dfc.png)